src/renderer/ducks/validator/actions.ts

```diff
@@ -83,14 +83,15 @@ export type SignedNewAttestation = (
     block: string,
     index: number,
     slot: number,
+    bits: string,
 ) => {
-    payload: {block: string; index: number; slot: number};
+    payload: {block: string; index: number; slot: number; bits: string};
     meta: string;
 };
 export const signedNewAttestation = createAction<SignedNewAttestation>(
     "validator/signedNewAttestation",
-    (publicKey: string, block: string, index: number, slot: number) => ({
-        payload: {block, index, slot},
+    (publicKey: string, block: string, index: number, slot: number, bits: string) => ({
+        payload: {block, index, slot, bits},
```

Comment:
you don't need bits here. YOu just need index of true bit in attestation (new attestation must have only 1 bit set - on index matching validator position in committee).